openslides_backend/action/motion_category/create_update_delete.py

```diff
@@ -0,0 +1,20 @@
+from ...models.motion_category import MotionCategory
+from ..action import register_action_set
+from ..action_set import ActionSet
+from ..default_schema import DefaultSchema
+
+
+@register_action_set("motion_category")
+class MotionCategoryActionSet(ActionSet):
+    """
+    Actions to create, update and delete meetings.
+    """
+
+    model = MotionCategory()
+    create_schema = DefaultSchema(MotionCategory()).get_create_schema(
+        properties=["name"], required_properties=["name"],
+    )
+    update_schema = DefaultSchema(MotionCategory()).get_update_schema(
+        properties=["name"]
```

Comment:
Please add the optional properties here that you added to the model.

Reply:
Done.